pkg/generate/ack/controller.go

```diff
@@ -93,6 +94,12 @@ var (
 		"GoCodeSetDeleteInput": func(r *ackmodel.CRD, sourceVarName string, targetVarName string, indentLevel int) string {
 			return code.SetSDK(r.Config(), r, ackmodel.OpTypeDelete, sourceVarName, targetVarName, indentLevel)
 		},
+		"GoCodeSetOperationStruct": func(r *ackmodel.CRD, targetFieldName string, targetVarName string, targetShapeRef *awssdkmodel.ShapeRef, sourceFieldPath string, sourceVarName string, indentLevel int) string {
```

Comment:
Can you call this `GoCodeSetSDKFromStruct` in order to match the name of the associated `pkg/generate/code` function? (Yes, I know there are other functions that do not match the name of the associated function, but I plan to clean those up in a separate PR...)